### ISSUE TITLE
tooltips are not localized

### ISSUE THREAD

**Issue by LuanVSO** [microsoft/terminal#5773]
# Environment

```none
Windows build number: 10.0.19041.0
Windows Terminal version (if applicable):0.11.1251.0

Any other software?
```

# Steps to reproduce

- set windows to a language other than English
- verify that win term changed it's language
![image](https://user-images.githubusercontent.com/43626415/81226601-77d30680-8fc1-11ea-9fac-58c9713072f6.png)

- hover mouse over a button on app bar

# Expected behavior

- tooltips are displayed in system language

# Actual behavior

- tooltips are displayed in English
![image](https://user-images.githubusercontent.com/43626415/81226585-71448f00-8fc1-11ea-8d44-555c7b2e7f47.png)

**Comment by DHowett-MSFT** [comment]
Is the rest of Terminal localized?

**Comment by LuanVSO** [comment]
yes

**Comment by DHowett-MSFT** [comment]
It looks like there was a delay in us receiving the translations for these values. Thanks for letting us know.

**Comment by LuanVSO** [comment]
this is fixed on v0.11.1333.0

**Comment by zadjii-msft** [comment]
Thanks for following up!
